fml/platform/android/message_loop_android.cc

```diff
@@ -93,4 +108,33 @@ void MessageLoopAndroid::OnEventFired() {
   }
 }
 
+bool MessageLoopAndroid::Register(JNIEnv* env) {
+  jclass clazz = env->FindClass("android/os/Looper");
+
+  if (clazz == nullptr) {
+    return false;
+  }
+
+  g_looper_class = new fml::jni::ScopedJavaGlobalRef<jclass>(env, clazz);
+
+  FML_CHECK(!g_looper_class->is_null());
+
+  g_looper_prepare_method_ =
+      env->GetStaticMethodID(g_looper_class->obj(), "prepare", "()V");
+  FML_CHECK(g_looper_prepare_method_ != nullptr);
+
+  g_looper_loop_method_ =
+      env->GetStaticMethodID(g_looper_class->obj(), "loop", "()V");
+  FML_CHECK(g_looper_loop_method_ != nullptr);
+
+  g_looper_my_looper_method_ = env->GetStaticMethodID(
+      g_looper_class->obj(), "myLooper", "()Landroid/os/Looper;");
+  FML_CHECK(g_looper_my_looper_method_ != nullptr);
+
+  g_looper_quit_method_ =
```

Comment:
There's no check for `g_looper_quit_method_` is `nullptr`. Is this intended?

Reply:
It's missed before, added now